Release/src/http/client/http_client_asio.cpp

```diff
@@ -1659,7 +1663,66 @@ pplx::task<http_response> asio_client::propagate(http_request request)
     // Asynchronously send the response with the HTTP client implementation.
     this->async_send_request(context);
 
-    return result_task;
+    return result_task.then(std::bind(&asio_client::maybe_redirect,
+                                      this,
+                                      std::placeholders::_1,
+                                      request,
+                                      std::vector<uri>()));
+}
+
+pplx::task<http_response> asio_client::maybe_redirect(http_response response, http_request request, std::vector<uri> redirected_uris) {
+    static const status_code found_status_code = 302;
```

Comment:
This constant is already defined here: `web::http::status_codes::Found`

Reply:
Yes. Also, the PR code comments refer to RFC 2068, which has been obsoleted twice-over, with [RFC 7231 section 6.4](https://tools.ietf.org/html/rfc7231#section-6.4) being the most recent spec I believe. This spec also removes the arbitrary maximum redirect count of 5 and explicitly recommends identifying cycles.